scripts/bump_deps.lua

```diff
@@ -70,9 +70,8 @@ local function get_archive_info(repo, ref)
   local archive_path = temp_dir .. '/' .. archive_name
   local archive_url = 'https://github.com/' .. repo .. '/archive/' .. archive_name
 
-  vim.fs.rm(archive_path, { force = true })
   run_die(
-    { 'curl', '-sL', archive_url, '-o', archive_path },
+    { 'curl', '-sfL', archive_url, '-o', archive_path },
```

Comment:
Probably good to add `--show-error`, too.

(*Ceterum censeo* that exactly this flagpalooza is why a `vim.net.download()` would be valuable.)

Reply:
Doesn't make a difference in the output. I think it's because we don't use `stderr` from `vim.system`.